layouts/events/list.html

```diff
@@ -12,11 +12,17 @@ <h1 class="page-header__title">{{ .Title }}</h1>
   {{ partial "calendar.html" . }}
 
   <div class="page-content">
-  {{ range .Paginator.Pages.ByDate.Reverse }}
-    {{ .Render "summary" }}
-  {{ end }}
+    {{ $ics := .AlternativeOutputFormats.Get "ics" }}
 
-  {{ partial "pagination.html" . }}
+    Subscribe to the calendar: <a href="{{ $ics.RelPermalink }}">{{ $ics.Permalink }}</a>.
```

Comment:
Nice idea!
I'd be tempted to move it somewhere a bit less conspicuous though (is there a common icon we can add in?). To me the page already feels like the "main" bit is at the bottom (the list of events), so I'm noticing that this pushes them down further.

Maybe this will go away if we have more events in the calender (and thus its purpose on the page is much more immediately obvious) and/or is something to address separately?

Reply:
It only pushes them down a single line, so I think that's fine. Perhaps something a little nicer would be better, but I wanted the primary showing to be the URL rather than a link to the ics file, as I _believe_ that if you import a file rather than link to the URL, it only adds events in that point in time, which defeats the point of this